src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java

```diff
@@ -20,18 +28,29 @@
     @Autowired
     private ZeebeClient zeebeClient;
 
+    @Autowired
+    private RestHighLevelClient esClient;
+
     @Override
     public void configure() {
 
+        from("rest:get:/es/health")
+                .id("es-test")
+                .log(LoggingLevel.INFO, "## Testing es connection")
+                .process(exchange -> {
+                    GetIndexRequest request = new GetIndexRequest("*");
+                    GetIndexResponse response = esClient.indices().get(request, RequestOptions.DEFAULT);
+                    JSONArray array = new JSONArray(response.getIndices());
+                    exchange.getMessage().setBody(array.toString());
```

Comment:
There can be quite a lot of indexes with the wildcard. No need to send the data in response. Send a generic message saying connection is successful and alive. It's suffice to check if there are no errors while calling ES.

Reply:
Yes, ideally we should be using Cluster Health APIs. For now I'll merge this but as a future improvement, we'll need to move the API to https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/java-rest-high-cluster-health.html